Update readme (#5)
DevExpress-Examples/blazor-grid-command-buttons-with-custom-icons: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T807225)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# Blazor Data Grid - Use icons instead of default command buttons

> At the moment, our DataGrid for Blazor doesn't support any API to customize command button appearance. However, you can use templates to change appearance of command buttons.

Reviewer: ```suggestion
Currently, our DataGrid for Blazor does not support any API to customize command button appearance. However, you can use templates instead.
```

![Data Grid with Custom Icons](images/datagrid-command-icons.png)

> It's possible to place any content into the command column's [CellTemplate](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGridCommandColumn.CellTemplate) or [HeaderCellTemplate](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGridCommandColumn.HeaderCellTemplate) template, for example, <button> or <a> HTML elements.

Reviewer: ```suggestion
You can add content to the command column's [CellTemplate](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGridCommandColumn.CellTemplate) or [HeaderCellTemplate](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxDataGridCommandColumn.HeaderCellTemplate) template, for example, <button> or <a> HTML elements.

```
